### PR TITLE
Feat: ESX Compatibility 

### PR DESCRIPTION
**Pull request by Ruqen**

### CODE REVIEW

File: server/sv_main.lua [hunk 1]
```diff
@@ -90,32 +96,34 @@ RegisterNetEvent('WP:getPlate', function(playerId, ped)
     if currentDistance <= maxDistance then
         TriggerClientEvent('WP:setPlate', playerId, originalNetId, originalPlate, fakePlate, 'fake')
     else
-        TriggerClientEvent('WC:notify', playerId, "error", 1000, nil, "Too far from vehicle.")
+        player.showNotification("Too far from vehicle.")
     end
 end)
 
 RegisterNetEvent('plateSuccess', function(cl_OriginalPlate, cl_FakePlate, plateType)
+    local source   = source
```
Comment (RockySouthpaw) [inline server/sv_main.lua:118]: Indent should be in line with source

File: server/sv_main.lua [hunk 2]
```diff
@@ -90,32 +96,34 @@ RegisterNetEvent('WP:getPlate', function(playerId, ped)
...
+    local ply   = ESX.GetPlayerFromId(source)
```
Comment (RockySouthpaw) [inline server/sv_main.lua]: Indent should be inline with source

Comment (Ruqen) [reply server/sv_main.lua]: yessir

**Comment by RockySouthpaw** [comment]
Looks good to me, will do a quick test and merge shortly

**Pull request merged**